.github/workflows/test_on_push.yaml

```diff
@@ -58,13 +58,13 @@ jobs:
           curl --fail -X PUT -F rockspec=@$ROCK_NAME-scm-1.rockspec \
             https://${{ secrets.ROCKS_USERNAME }}:${{ secrets.ROCKS_PASSWORD }}@${{ secrets.ROCKS_SERVER }}
       - name: Push release rockspec
-        if: github.event_name == 'push' && github.ref == 'refs/tags/'
```

Comment:
Why did you removed `event_name` check?

Reply:
`startsWith(github.ref, 'refs/tags')` is enough. Also I don't know alternative ways to create a tag